pkg/analyze/host_analyzer.go

```diff
@@ -5,7 +5,7 @@ import troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubl
 type HostAnalyzer interface {
 	Title() string
 	IsExcluded() (bool, error)
-	Analyze(getFile func(string) ([]byte, error)) ([]*AnalyzeResult, error)
+	Analyze(getFile func(string) ([]byte, error), findFiles getChildCollectedFileContents) ([]*AnalyzeResult, error)
```

Comment:
Adding a new function parameter to this function has led to a lot of function signature changes in all host analysers, hence the many changed files in the PR